cmd/diff.go

```diff
@@ -119,25 +114,5 @@ func (o *DiffOptions) Run() (err error) {
 		return
 	}
 
-	// TODO (b5): this reading from a package variable is pretty hacky :/
-	if color.NoColor {
-		stats = deepdiff.FormatPrettyStats(res.Stat)
-		if !o.Summary {
-			text, err = deepdiff.FormatPretty(res.Diff)
-			if err != nil {
-				return err
-			}
-		}
-	} else {
-		stats = deepdiff.FormatPrettyStatsColor(res.Stat)
-		if !o.Summary {
-			text, err = deepdiff.FormatPrettyColor(res.Diff)
-			if err != nil {
-				return err
-			}
-		}
-	}
-	buf := bytes.NewBuffer([]byte(stats + "\n" + text))
-	printToPager(o.Out, buf)
-	return
+	return printDiff(o.Out, res, o.Summary)
```

Comment:
Looks like this is just encapsulating the diff printing code in a function with no other changes related to coloring? Just making sure I’m not missing anything.

Reply:
yep exactly. This just moves the `print` things from 3 files to 2. I'm trying to parse out my commit history better. It's a work in progress 😄